test/test_initial.py

```diff
@@ -37,6 +37,28 @@ def test_crosstab_without_suppression(data):
     assert 48 == output.output[0]["R/G"].sum()
 
 
+def test_crosstab_multiple_aggregate_function(data, acro):
+    """Crosstab with multiple agg funcs."""
+    acro = ACRO(suppress=False)
+
+    _ = acro.crosstab(
+        data.year, data.grant_type, values=data.inc_grants, aggfunc=["mean", "std"]
+    )
+    output = acro.results.get_index(0)
+    correct_summary: str = (
+        "fail; threshold: 12 cells may need suppressing;"
+        " p-ratio: 2 cells may need suppressing; "
+        "nk-rule: 2 cells may need suppressing; "
+    )
+    assert (
+        output.summary == correct_summary
+    ), f"\n{output.summary}\n should be \n{output.summary}\n"
```

Comment:
should be `correct_summary`?

Reply:
pytest output tells you the values though anyway if it fails, so not sure why it's needed after the assert?

Reply:
'cos running pytest in my laptop gives me a fairly messy output and crops long lines.
That's why I used an f-string with newlines in it so it's easier to look at them aligned the same.
Not a big deal.

Reply:
but yes you are right about the f-string. Will change

Reply:
@rpreen don't understand GitHub :( I've pushed a new commit, and it shows below, and if I look. at the files through the code viewer. assume this is ok

Reply:
something strange is definitely happening, i got notification of the push but it's not showing anywhere

Reply:
and the file looks right if you click to open it in codespace on the web page.